frontend/src/routes/(app)/dashboard/+page.svelte

```diff
@@ -1,104 +1,43 @@
 <script lang="ts">
 	import { _ } from 'svelte-i18n';
-	import { Plus, Trash2 } from 'lucide-svelte';
-	import ConfirmDialog from '$lib/components/dialogs/ConfirmDialog.svelte';
-	import { deleteLearningKit, getAllLearningKits } from '$lib/api/collections/learningKit';
-	import { createMutation, createQuery, useQueryClient } from '@tanstack/svelte-query';
+	import { ChevronRight } from 'lucide-svelte';
+	import { getLatestFiveLearningKits } from '$lib/api/collections/learningKit';
+	import { createQuery } from '@tanstack/svelte-query';
 	import { api } from '$lib/api/apiClient.js';
 	import ErrorIllustration from '$lib/components/ErrorIllustration.svelte';
+	import AddLearningKit from '$lib/components/learningkit/AddLearningKit.svelte';
+	import LearningKit from '$lib/components/learningkit/LearningKit.svelte';
+	import PlaceholderLearningKit from '$lib/components/learningkit/PlaceholderLearningKit.svelte';
+	import DashboardBase from '$lib/components/DashboardBase.svelte';
 
-	const client = useQueryClient();
-
+	const queryKey = 'latest-learning-kits-list';
```

Comment:
You could add a second script with module attribute in this file and export somnething like LATEST_LEARNING_KIT_LIST instead of passing it via prop